$NetBSD: patch-CMakeLists.txt,v 1.1 2020/07/26 07:54:45 maya Exp $

* fix RPATH to be absolute path
* always install woff2* binaries.

--- CMakeLists.txt.orig	2017-11-13 18:31:28.000000000 +0000
+++ CMakeLists.txt
@@ -26,9 +26,9 @@ set(WOFF2_VERSION 1.0.2)
 set(CMAKE_SKIP_BUILD_RPATH FALSE)
 set(CMAKE_BUILD_WITH_INSTALL_RPATH FALSE)
 set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
-list(FIND CMAKE_PLATFORM_IMPLICIT_LINK_DIRECTORIES "${CMAKE_INSTALL_LIBDIR}" isSystemDir)
+list(FIND CMAKE_PLATFORM_IMPLICIT_LINK_DIRECTORIES "${CMAKE_INSTALL_FULL_LIBDIR}" isSystemDir)
 if ("${isSystemDir}" STREQUAL "-1")
-  set(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_LIBDIR}")
+  set(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_FULL_LIBDIR}")
 endif()
 
 # Find Brotli dependencies
@@ -260,12 +260,10 @@ generate_pkg_config ("${CMAKE_CURRENT_BI
   LIBRARIES woff2enc)
 
 # Installation
-if (NOT BUILD_SHARED_LIBS)
   install(
     TARGETS woff2_decompress woff2_compress woff2_info
     RUNTIME DESTINATION "${CMAKE_INSTALL_BINDIR}"
   )
-endif()
 
 install(
   TARGETS woff2common woff2dec woff2enc
